src/server/process_request.c

```diff
@@ -569,9 +570,9 @@ process_request(int sfds)
 
 	access_by_krb = 0;
 
-	/* FIXME: Do we need realm check for all auth ? */
 #if defined(PBS_SECURITY) && (PBS_SECURITY == KRB5)
-	if (conn->cn_credid != NULL &&
+	if (request->rq_type != PBS_BATCH_Connect &&
```

Comment:
Why we don't want it to verify for PBS_BATCH_Connect ?
Because it verifies whether server attribute acl_krb_realm_enable set.

Reply:
Removed. This was useless anyway. The `conn->cn_credid` and other `conn` attributes are not set in the `PBS_BATCH_Connect`. Will be set after authentication.